### PR TITLE
[FEAT] Reference demo in the example html home page

### LINKED ISSUE #87: [FEAT] Reference the demo in the examples html home page
As we are going to only keep the latest demo version (#86)

- [x] Create a new section on top of the page and place title and a card for the demo
- [x] Document how to get bpmn files (miwg-test-suite, ...) for local file loading on top of the page
- [x] Remove reference to 'how to get files' from the README (as already doc in the example html home page)
- [x] Rework the readme and keep only one section for live environment: no more demo/examples split
- [x] Update the image of the README as it is no up-to-date: use a generic image (B.2.0 rendering for instance) to avoid changes when the home page changes

### PR DESCRIPTION
**Pull request by tbouffard**
Closes #87

**Live environment of examples**

https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/87-demo_ref_in_example_html_home_page/examples/index.html

### CODE REVIEW

**Comment by tbouffard** [comment]
I am not super happy with the way the 'list of resources available to get bpmn diagrams file' is rendered. This block of text make the beginning of the page messy.
I tried to put it after a h4 section, but the result is not great (see below).

image


Feel free to try something else to improve.

File: examples/index.html [hunk 1]
```diff
@@ -47,9 +47,47 @@
...
+                        </div>
+                        <div class="card-header">
+                            <div class="card-title h5">Load and Navigation demo</div>
+                            <div class="card-subtitle text-gray">show most of features provided by <i>bpmn-visualization</i> about
+                                BPMN rendering and diagram navigation when loading BPMN local files
+                            </div>
+                        </div>
+                        <div class="card-body"></div>
+                        <div class="card-footer">
+                            <a href="../demo/index.html" target="_blank" class="btn btn-primary">go to demo</a>
```
Comment (csouchet) [inline examples/index.html:87]: I thought we keep only the last demo as example. But it is ok to keep like this for nom 🙂

**Review by csouchet — APPROVED** [review]

File: examples/index.html (hunk 1 above)
Comment (tbouffard) [reply examples/index.html:87]: Keeping the last demo version is managed independently in #91 because it depends on #88 first.

**Comment by tbouffard** [comment]
@csouchet what do you of the section about 'how to get local BPMN diagram'?

**Review by aibcmars — APPROVED** [review]
LGTM

**Comment by csouchet** [comment]
> @csouchet what do you think of the section about `how to get local BPMN diagram`?

It's Ok for now. But I think we can improve it (but I don't know how ^^)

**Pull request merged**